content/working-with-azure-key-vault-in-csharp/index.md

```diff
@@ -117,11 +117,11 @@ To enable the system-assigned managed identity for the Azure Web App we just cre
 
 Figure 3
 
-## **Create and Configure Azure Key Vault**
+## Create and Configure Azure Key Vault
 
 In this section we’ll examine how we can create and configure an Azure key Vault instance.
 
-### **Create a new Azure Key Vault Instance**
+### Create a new Azure Key Vault Instance
 
 We'll now create an Azure Key Vault in the Azure Portal and then add a secret to it. To create a new Azure Key Vault instance, navigate to the Azure Portal and follow the steps mentioned below:
```

Comment:
Changes: follow the steps below: (deleted: `mentioned`)

Reply:
Changes: In the Create key vault screen`,` provide the name, subscription, resource group name, location`,` and leave the other options to their default values.